egs/chime4/asr1_mc/conf/chime4.cfg

```diff
@@ -0,0 +1,50 @@
+#BeamformIt sample configuration file for AMI data (http://groups.inf.ed.ac.uk/ami/download/)
```

Comment:
Does it require beamformit?
I think you can remove this and also beamformit parts in `run.sh`.

Reply:
I do not change chime4.cfg from asr1 setup.

Reply:
My point is whether you require BeamformIt in this recipe (`run.sh`).
If you need BeamformIt, you can keep this, but otherwise, you can remove this.

Reply:
I know that you call BeamformIt in `run.sh`. Again if you need the BeamformIt processing, you can keep this file and the BeamformIt processing in `run.sh`. Otherwise remove both.

Reply:
I see. I think I can remove BeamformIt. I just called it in run.sh because it was called in asr1's run.sh.